script/cmtool.bat

```diff
@@ -102,7 +102,7 @@ set url="https://omnitruck.chef.io/%OMNITRUCK_CHANNEL%/%OMNITRUCK_PRODUCT%/metad
 set filename="%TEMP%\omnitruck.txt"
 
 echo ==^> Using Chef Omnitruck API URL: !url!
-powershell -command "(New-Object System.Net.WebClient).DownloadFile('!url!', '!filename!')"
+call "%SystemRoot%\_download.cmd" !url! !filename!
```

Comment:
As per my comment for `floppy/bitvisessh.bat`, it's likely better that we check for the existence of `_download.cmd` at the top of the script and emit the error there if you chose to not do them in the conditional.